ui/src/clockface/components/resource_list/ResourceCard.tsx

```diff
@@ -0,0 +1,132 @@
+// Libraries
+import React, {PureComponent} from 'react'
+import {Link} from 'react-router'
+import moment from 'moment'
+
+// Types
+import {Organization} from 'src/types/v2'
+
+// Constants
+import {UPDATED_AT_TIME_FORMAT} from 'src/dashboards/constants'
+
+interface PassedProps {
+  name: () => JSX.Element
+  description?: () => JSX.Element
+  updatedAt?: string
+  owner?: Organization
+  labels?: () => JSX.Element
+  meta1?: () => JSX.Element
```

Comment:
Are these for resource metadata? The meta1 & meta2 props are ambiguous and limiting.

Reply:
Refactored to be a single prop `metaData: () => JSX.Element[]`